src/lesson1/Colors.java

```diff
@@ -0,0 +1,9 @@
+package lesson1;
+
+public enum Colors {
+    BLACK,
+    WHITE,
+    BLACK_AND_WHITE,
+    RED,
+    TRICOLOR,
```

Comment:
I think main theme of your task include this Enum to your class Cat like a field and try to manupalaite with it

Reply:
no its not(yet)